pkgs/servers/mastodon/source-unpatched.nix

```diff
@@ -0,0 +1,9 @@
+let
+  pkgs = import <nixpkgs> {};
```

Comment:
I am not using `<nixpkgs>` variable at all (there is `<nixpkgs-booted>`, `<nixpkgs-current>` and `<nixpkgs-next>` instead), so this pattern does not work.
Probably better to change to
```
{ fetchFromGitHub }:
fetchFromGitHub {
 ...
}
```
and `callPackage ./source-unpatched.nix {};` instead of `import ./source-unpatched.nix;`;
It is even shorter.

Reply:
I did pretty much the same thing in #78810 to remove the reference to `<nixpkgs>`.